.travis.yml

```diff
@@ -6,6 +6,9 @@ php:
     - 5.5
     - 5.6
     - 7.0
```

Comment:
Could you maybe remove the 5.x jobs? I don't think PHP5 is still a thing anymore.

Reply:
@weierophinney, that's right.

It's time to drop the `php-5.x` supports and release the latest stable version.